GameServerLib/Game.cs

```diff
@@ -104,7 +104,7 @@ public class Game : IGame
         /// <summary>
         /// Interface for all map properties used for the game.
```

Comment:
Update summary `Contains all map related game settings such as collision handler, navigation grid, announcer events, and map properties. Doubles as a Handler/Manager for all MapScripts.`